apps/Launcher/syncwidget.cpp

```diff
@@ -225,8 +225,11 @@ void SyncWidget::setSceneFiles(QMap<QString, QString> sceneFiles) {
         const QString& sceneName = keys[i];
 
         QCheckBox* checkbox = new QCheckBox(sceneName);
-        checkbox->setChecked(true);
-
+        checkbox->setChecked(false);
+        QString defaultName = "default";
+        if(QString::compare(defaultName, sceneName, Qt::CaseInsensitive)==0){
```

Comment:
1. Instead of `QString::compare`, it's better to use the overloaded `==` operator:  `defaultName == sceneName`.  The cases should all be lowercase anyway.

2. Coding style:
```
if (defaultName == sceneName) {
```